datafusion/core/Cargo.toml

```diff
@@ -61,13 +61,21 @@ default = [
     "unicode_expressions",
     "compression",
     "parquet",
+    "parquet_encryption",
```

Comment:
Should this really be a default feature? It's still marked experimental in arrow-rs.

Reply:
Probably a question for @alamb

Reply:
I would personally prefer if it was not part of the default features but given it is already on by default on main, I think it is fine to leave it on by default in this PR

We can discuss disabling the feature by default as a follow on ticket / PR perhaps

Here are some notes I wrote about how to make it a non default feature:

1. Don't add it to the default features
1. Update the docs to mention the config setting requires the `parquet_encryption` feature -- https://github.com/apache/datafusion/blob/d66d6b9eb83a4643bc8d1ee9c2da7f0fa644c0cf/datafusion/common/src/config.rs#L1839
2. Enable it in datafusion-cli (add `parquet_encryption` in https://github.com/apache/datafusion/blob/ca16255e725bf6676a8ddd4b9948496d99b5bd88/datafusion-cli/Cargo.toml#L45-L44)

Reply:
I think this would make for a good followup PR. Thanks for the helpful notes.

Reply:
I think this would make for a good follow-up PR. Thanks for the helpful notes.